provision.sh

```diff
@@ -7,10 +7,16 @@ echo "---------------------------------------------"
 sudo apt-get update
 sudo apt-get install -y python-pip
 sudo apt-get install -y git
+sudo apt-get install -y nano
 sudo pip install virtualenvwrapper
-sudo wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
-cd /vagrant && virtualenv --python=python3.4 myvenv && source myvenv/bin/activate && pip install django==1.9.5 && django-admin startproject myproject && pip install django-widget-tweaks
 
 echo "---------------------------------------------"
-echo " Finished."
+echo "Environment setup complete!"
+echo "---------------------------------------------"
+
 echo "---------------------------------------------"
+echo "Installing django"
+echo "---------------------------------------------"
+sudo mkdir /vagrant
```

Comment:
Can you please check if there is any synced directories in your vagrant VM. This is extremely important and would defeat the point of using vagrant as dev VM if there is no synced folder. In case there is none, you can add it by writing a few lines in Vagrantfile. There should be some articles on it on SO.